src/rating-summary/styles.module.scss

```diff
@@ -0,0 +1,70 @@
+@keyframes widthAnimation {
+	from {
+		width: 0px;
+	}
+	to {
+		width: 100%;
+	}
+}
+.ratingsWrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	justify-content: flex-start;
+	width: 100%;
+	row-gap: 2px;
+	font-family: inherit;
+	.ratingWrapper {
+		display: flex;
+		align-items: center;
+		column-gap: 8px;
+		width: 100%;
+		.label {
+			display: flex;
+			align-items: center;
+			font-size: 15px;
+			line-height: 17px;
+			color: #919191;
+			min-width: max-content;
+			float: left;
+		}
+		.barWrapper {
+			max-width: 100%;
+			.barContainer {
+				height: 20px;
+				overflow-x: visible;
+				display: flex;
+				align-items: center;
+				width: 100%;
+				.countContainer {
+					font-size: 11px;
+					line-height: 17px;
+					background-color: transparent;
+					color: #65695b;
+					padding-left: 8px;
+				}
+			}
+			.transitions {
+				animation-name: widthAnimation;
+				animation-duration: 300ms;
+				animation-timing-function: ease-in-out;
+				animation-iteration-count: 1;
+			}
+		}
+		.five {
+			background-color: #9fc05a;
```

Comment:
Are we giving an option to give different colour for each rating?

Reply:
user can chnage chart colour using the style prop, not giving any specific prop for the chart colours as of now

Reply:
updated with chart color customization props as well